tardis/transport/montecarlo/packet_trackers.py

```diff
@@ -47,48 +60,53 @@ class RPacketTracker(object):
             Type of interaction the rpacket undergoes
         num_interactions : int
             Internal counter for the interactions that a particular RPacket undergoes
+        extend_factor : int
+            The factor by which to extend the properties array when the size limit is reached
     """
 
     def __init__(self, length):
-        self.length = length
```

Comment:
I removed this length variable, since If I kept this length, I would have to add another length variable for every track_interaction function.